### PR TITLE
[zh-cn]: update the translation of Atomics.exchange() and compareExchange()

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

Same as the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

https://github.com/mdn/content/pull/31315
#17175

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/atomics/compareexchange/index.md [hunk 1]
```diff
@@ -1,49 +1,54 @@
 ---
 title: Atomics.compareExchange()
 slug: Web/JavaScript/Reference/Global_Objects/Atomics/compareExchange
+l10n:
+  sourceCommit: a92a2bb31cf5d79808878701f0344a4eabf12963
```
Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/global_objects/atomics/compareexchange/index.md:5]: 更新一下 sourceCommit

```suggestion
  sourceCommit: ec9fab3524d17932694856be045c9e50168821fd
```

File: files/zh-cn/web/javascript/reference/global_objects/atomics/compareexchange/index.md [hunk 2]
```diff
@@ -1,49 +1,54 @@
...
 ---
 
-{{JSRef}} {{SeeCompatTable}}
+{{JSRef}}
 
-**`Atomics.compareExchange()`** 静态方法会在数组的值与期望值相等的时候，将给定的替换值替换掉数组上的值，然后返回旧值。此原子操作保证在写上修改的值之前不会发生其他写操作。
+如果给定的预期值等于旧值，**`Atomics.exchange()`** 静态方法会用给定的值替换数组上指定位置的值，并返回该位置的旧值。此原子操作保证在修改后的值写回之前不会发生其他写操作。
```
Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/global_objects/atomics/compareexchange/index.md:10]: 不只是返回旧值
> It returns the old value at that position whether it was equal to the expected value or not.

**Review by jasonren0403 — COMMENTED** [review]
suggestions

File: files/zh-cn/web/javascript/reference/global_objects/atomics/compareexchange/index.md [hunk 3]
```diff
@@ -1,49 +1,54 @@
...
 slug: Web/JavaScript/Reference/Global_Objects/Atomics/compareExchange
+l10n:
+  sourceCommit: ec9fab3524d17932694856be045c9e50168821fd
 ---
 
-{{JSRef}} {{SeeCompatTable}}
+{{JSRef}}
 
-**`Atomics.compareExchange()`** 静态方法会在数组的值与期望值相等的时候，将给定的替换值替换掉数组上的值，然后返回旧值。此原子操作保证在写上修改的值之前不会发生其他写操作。
+如果给定的预期值等于旧值，**`Atomics.exchange()`** 静态方法会用给定的值替换数组上指定位置的值，并返回该位置的旧值。此原子操作保证在修改后的值写回之前不会发生其他写操作。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/atomics/compareexchange/index.md:10]:
```suggestion
如果给定的预期值等于旧值，**`Atomics.exchange()`** 静态方法会用给定的值替换数组上指定位置的值，并返回该位置的旧值，无论其是否等于预期的值。此原子操作保证在修改后的值写回之前不会发生其他写操作。
```

**Review by jasonren0403 — APPROVED** [review]
LGTM, Thanks @JasonLamv-t

**Pull request merged**